cuda_bindings/setup.py

```diff
@@ -345,7 +345,7 @@ def do_cythonize(extensions):
     (["cuda/bindings/_lib/cyruntime/utils.pyx"], None),
     (["cuda/bindings/utils/*.pyx"], None),
     # public
-    *(([f], None) for f in cuda_bindings_files),
+    *(([f, "cuda/bindings/_lib/param_packer.cpp"], None) for f in cuda_bindings_files),
```

Comment:
btw previously we found this pattern problematic because the same cpp file could be compiled multiple times and MSVC fails randomly when we turn on parallel build (https://github.com/NVIDIA/cuda-python/issues/271). We should stay away form this